### PR TITLE
FOUR-19328: Enable the order desc/asc "Tasks" tab

### PR DESCRIPTION
**Pull request by henryjonathanquispe**
## Summary
Enable the order desc/asc "Tasks" tab

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-19328

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

ci:next

### CODE REVIEW

File: resources/jscomposition/cases/casesDetail/api/index.js [hunk 1]
```diff
@@ -22,4 +22,90 @@ const getData = async () => {
...
+    from: 1,
+    last_page: 1,
+    links: [
+      {
+        url: null,
+        label: "&laquo; Previous",
+        active: false,
+      },
+      {
+        url: "http:\/\/localhost:8092\/api\/1.0\/tasks-by-case?page=1",
```
Comment (pmPaulis) [inline resources/jscomposition/cases/casesDetail/api/index.js:70]: The URL is harcoded localhost:8092

File: resources/jscomposition/cases/casesDetail/api/index.js [hunk 2]
```diff
@@ -22,4 +22,90 @@ const getData = async () => {
...
+        label: "1",
+        active: true,
+      },
+      {
+        url: null,
+        label: "Next &raquo;",
+        active: false,
+      },
+    ],
+    path: "http:\/\/localhost:8092\/api\/1.0\/tasks-by-case",
```
Comment (pmPaulis) [inline resources/jscomposition/cases/casesDetail/api/index.js:80]: The URL is harcoded localhost:8092

**Review by pmPaulis — CHANGES_REQUESTED** [review]

File: resources/jscomposition/system/table/SortTable.vue [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    :data="data"
+    class="tw-grow tw-overflow-y-scroll">
+    <template
+      v-for="(column, index) in columns"
+      #[`theader-filter-${column.field}`]>
+      <SortableFilter
+        v-if="column.filter"
+        :key="`sortable-${index}`"
+        :state="indexFilter == index? 'asc': ''"
+
```
Comment (fagubla) [inline resources/jscomposition/system/table/SortTable.vue:14]: remove extra line

File: resources/jscomposition/system/table/SortTable.vue [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    :columns="columns"
+    :data="data"
+    class="tw-grow tw-overflow-y-scroll">
+    <template
+      v-for="(column, index) in columns"
+      #[`theader-filter-${column.field}`]>
+      <SortableFilter
+        v-if="column.filter"
+        :key="`sortable-${index}`"
+        :state="indexFilter == index? 'asc': ''"
```
Comment (fagubla) [inline resources/jscomposition/system/table/SortTable.vue:13]: add an space after index

File: resources/jscomposition/system/table/filter/sortableFilter/SortableFilter.vue [hunk 5]
```diff
@@ -0,0 +1,43 @@
+<template>
+  <div
+    :class="{
+      'tw-text-blue-500': order
+    }"
```
Comment (fagubla) [inline resources/jscomposition/system/table/filter/sortableFilter/SortableFilter.vue:5]: this would be in a single line

**Review by fagubla — CHANGES_REQUESTED** [review]

File: resources/jscomposition/base/table/THeader.vue [hunk 6]
```diff
@@ -59,3 +56,15 @@ export default defineComponent({
   },
 });
 </script>
+<style>
```
Comment (fagubla) [inline resources/jscomposition/base/table/THeader.vue:59]: styles must be scoped

**Review by fagubla — CHANGES_REQUESTED** [review]

**Review by fagubla — APPROVED** [review]

**Pull request merged**